foldamers/cg_model/cgmodel.py

```diff
@@ -875,45 +750,38 @@ def get_particle_type(self, particle_index, particle_name=None):
           :param particle_index: Index of the particle for which we would like to determine the type
           :type particle_index: int
 
-          :param particle_name: Name of the particle that we would like to "type".
-          :type particle_name: str
-
           :returns: 
              - particle_type (str) - 'backbone' or 'sidechain'
 
           """
-        if particle_name == None:
-            particle_name = self.particle_list[particle_index]
-
-        particle_type = None
-
-        if "X" in particle_name:
+        if self.particle_list[particle_index]["type"] == "backbone":
             particle_type = "backbone"
-        if "A" in particle_name:
+        elif self.particle_list[particle_index]["type"] == "sidechain":
             particle_type = "sidechain"
-
-        if particle_type == None:
-            print(
-                "ERROR: The particle type definition could not be found for: " + str(particle_name)
-            )
-            print(
-                "By default, 'backbone' particles are expected to have the following naming conventions:"
-            )
-            print("'X'+int (ie: X1, X2, etc.)")
-            print("\n")
-            print("'sidechain' particles are expected to have the following naming conventions:")
-            print("'A'+int (ie: A1, A2, etc.)")
-            print("If you built your model using positions read from a PDB file,")
-            print(
-                "try renaming the particles in the PDB file to conform to these naming conventions."
-            )
-            exit()
+        else:
+            print(f"ERROR: The particle type definition could not be found for {particle_index}")
 
         return particle_type
 
+    def get_particle_monomer(self, particle_index):
+        """
+          Indicates if a particle is a backbone bead or a sidechain bead
```

Comment:
This description wasn't changed from get_particle_type

Reply:
OK fixed.